fixmydjango/urls.py

```diff
@@ -31,6 +34,23 @@ def activate(request, uid, token):
 
 
 urlpatterns = [
+    path(
+        'openapi',
+        get_schema_view(
+            title="FMC API",
+            description="The API documentation for fixmy.frontend, fixmy.radparken ....",
+            version="1.0.0",
+        ),
```

Comment:
For using the live editor, I think we would need also the `url` here of the api.
See also https://www.django-rest-framework.org/api-guide/schemas/#generating-an-openapi-schema